src/cljs/main/broadfcui/header.cljs

```diff
@@ -41,7 +41,8 @@
    {:label [:div {:style {:borderRadius 2
                           :backgroundColor (:background-light style/colors)
                           :color "#000" :textDecoration "none"
-                          :padding "0.5rem" :border style/standard-line}}
+                          :padding "0.5rem" :border style/standard-line}
+                  :data-test-id "accountDropdown"}
```

Comment:
Isaac is right. I'd prefer "account-dropdown" unless there is a particular reason to use camel case for these.